### PR TITLE
Ma ac firebase

### PR DESCRIPTION
**Pull request by alxmcr**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

 As a component, I want to be able to read from and write to the Firestore database so that users can persist information.

## Related Issue

#1 

## Acceptance Criteria

- [x] The following have been added as project dependencies: firebase & react-firebase-hooks
- [x] Make a change in the Firestore database and it shows up in the app
- [x] Make a change in the app and it shows up in the Firestore database

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|   ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

image

### After

image

## Testing Steps / QA Criteria

1. Then `npm `to `install `the newly added dependencies locally and `npm start` to launch the app.
2.  Press the button "+" multiple times then sales value increases one by one (For example: 0, 1, 2, 3, ... n). Check the value in Firebase.
3. Press the button "reset" then the sales value resets to zero. Check the value in Firebase.

### CODE REVIEW

**Comment by alxmcr** [comment]
Hi @Lukaina and @CrisRuedaP! 😊

Please, we would appreciate it a lot, when you have a free time, on your agenda 🙏. 

Could you review the work we did @macosta08 and me 😄

Any suggestion, correction or comment are totally welcome! 😃

**Review by Lukaina — COMMENTED** [review]
Hello, 

After reviewing the work you did, we want to congratulate you, everything is in order and we feel that the code is easy to interpret. Thanks for the work!

**Comment by alxmcr** [comment]
Thank you so much, Veronica! 😄

**Comment by macosta08** [comment]
Thank you Veronica, for your opinion I appreciated it very much.

**Review by Lukaina — APPROVED** [review]

**Review by CrisRuedaP — APPROVED** [review]
You did a great job @macosta08 and @alxmcr. 

Well done. 👏👏👏

Approved. ✔

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import CounterList from './components/couterList';
```
Comment (gmzjuliana) [inline src/App.js]: there's a small typo here, would you be up for fixing it? couterList => counterList

File: src/components/counter/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    var counterRef = db.collection('counters').doc(id);
+
+    counterRef
+      .update({
+        value: currentValue,
+      })
+      .then(function () {
+        console.log('Document successfully updated!');
+      })
+      .catch(function (error) {
```
Comment (gmzjuliana) [inline src/components/counter/index.js:19]: nice! love seeing a good catch for errors

**Review by gmzjuliana — APPROVED** [review]
great work y'all! I left a small comment on a typo in the filename for the counterList but it's not blocking 😁

**Review by indiragp — APPROVED** [review]
Good Job! I agree with the comment that Juliana made about the typo in a file name. 
Besides that I would suggest taking a look at this blog: https://medium.com/hackernoon/absolute-imports-with-create-react-app-4c6cfb66c35d and trying to clean up a bit your imports :)

File: src/App.js (hunk 1 above)
Comment (alxmcr) [reply src/App.js]: Thank you @gmzjuliana! We'll review it!

File: src/components/counter/index.js (hunk 2 above)
Comment (alxmcr) [reply src/components/counter/index.js:19]: Thank you for your great comments! 😊

**Comment by alxmcr** [comment]
Thank you @gmzjuliana and @indiragp for your review!
 
We resolved the typo and improved imports 😄.  

Could you please review it again 🙏?  We'll appreciate it!

**Pull request merged**